en/micro-integrator/docs/references/connectors/servicenow-connector/servicenow-connector-example.md

```diff
@@ -0,0 +1,256 @@
+# ServiceNow Connector Example
+
+ServiceNow is an application platform as a service, which is a cloud-based computing model that provides the infrastructure needed to develop, run, and manage applications. It offers activities of an organization such as data collection, storage, workflow automation, and reporting through a single user interface. This software as a service (SaaS) platform contains a number of modular applications that can vary by instance and user. It focuses on service-orientation toward the tasks, activities, and processes.
+
+The WSO2 ServiceNow connector allows you to access the ServiceNow REST API through WSO2 EI. Using ServiceNow connector you can work with Aggregate API, Import Set API and Table API in ServiceNow. You can further read about ServiceNow REST APIs from [here.](https://developer.servicenow.com/dev.do#!/reference/api/orlando/rest/c_TableAPI)
+
+## What you'll build
+
+This example explains how to use ServiceNow Connector to create records in a table and retrieve its information. Assume your organization uses ServiceNow support and you need to create an incident. In order to do that we can use the tableAPI which is a Rest API. This can be easily done using WSO2 ServiceNow connector. Whenever integration needs to raise an incident at ServiceNow, above EI API can be called with required information.
+
+It will have two HTTP API resources, which are `postRecord` and `readRecord`. 
+
+<img src="/assets/img/connectors/serviceNow.png" title="ServiceNow Scenario" width="800" alt="ServiceNow Scenario"/>
+
+* `/postRecord`: It creates a new record in the existing incident table in ServiceNow instance 
+
+* `/readRecord `: It reads the detailed information about the created incident record in the incident table.
+
+
+## Setting up the environment 
+
+Please follow the steps mentioned at [Setting up Servicenow Instance](settingup-servicenow-instance.md) document in order to create a Servicenow Instance and obtain the credentials. 
+Keep them saved to be used in the next steps.  
+
+## Configure the connector in WSO2 Integration Studio
+
+Follow these steps to set up the Integration Project and the Connector Exporter Project. 
+
+{!references/connectors/importing-connector-to-integration-studio.md!} 
+
+1. First let's create postRecord sequence and ReadRecord sequences. Right click on the created ESB Solution Project and select, -> **New** -> **Sequence** to create the Sequence. 
```

Comment:
Need to update this. It is called Integration Project in the latest tooling pack.

Reply:
Done